docker/wedlocks.dockerfile

```diff
@@ -2,7 +2,6 @@ FROM unlock-core
 
 # Dependencies for wedlocks
 RUN mkdir /home/unlock/wedlocks
-COPY --chown=node wedlocks/package-lock.json /home/unlock/wedlocks/.
```

Comment:
Same comment as in the other PR, I think this branch is missing the `yarn.lock` files

Reply:
Even if these are ones that slipped through the last PR, we should replace `package-lock.json` with the associated `yarn.lock`, because without it we may not always get the same set of deps on each build.